doc/WebSite/Entity-History.md

```diff
@@ -156,12 +156,12 @@ You can use the **IEntityChangeSetReasonProvider.Use(...)** method as shown belo
     }
 
 The Use method returns an IDisposable and it **must be disposed**. Once the return
-value is disposed, Reason is automatically restored to the previous value.
+value is disposed, the Reason is automatically restored to the previous value.
```

Comment:
Again, "the Use **method**" reads well.

Here, "the Reason" is passable as it's been referred to a number of times in this section **and** because it's linked to "the previous value".